ext/standard/tests/strings/fprintf_variation_002.phpt

```diff
@@ -3,7 +3,7 @@ Test fprintf() function (variation - 2)
 --FILE--
 <?php
 
-$int_variation = array( "%d", "%-d", "%+d", "%7.2d", "%-7.2d", "%07.2d", "%-07.2d", "%'#7.2d" );
+$int_variation = array( '"%d"', '"%-d"', '"%+d"', '"%7.2d"', '"%-7.2d"', '"%07.2d"', '"%-07.2d"', '"%\'#7.2d"' );
```

Comment:
Why this change?

Reply:
These changes were made because of trailing whitespaces, I think they are useful to keep but probably as a separate PR/commit.